src/test.js

```diff
@@ -4,350 +4,329 @@ const autodux = require('./');
-      { type: 'counter/decrement', payload: undefined },
-      { type: 'counter/multiply', payload: 2 }
-    ];
+    const { actions } = createCounterDux();
 
     assert({
-      given: 'autodux is called with args',
-      should: 'produce correct action objects',
-      actual,
-      expected
+      given: "'autodux' is called with 'actions'",
+      should: 'contain action creators that return correct action objects',
+      actual: [
+        actions.increment(),
+        actions.decrement(),
+        actions.multiply({ by: 2 }),
+        actions.randomize({ min: 0, max: 5 })
+      ],
+      expected: [
+        { type: 'counter/increment', payload: undefined },
+        { type: 'counter/decrement', payload: undefined },
+        { type: 'counter/multiply', payload: 2 },
+        { type: 'counter/randomize', payload: { min: 0, max: 5 } }
+      ]
     });
   }
 
   {
     const {
-      actions: { setCounter, increment, decrement, multiply }
-    } = createDux();
-
-    const actual = [
-      setCounter.type,
-      increment.type,
-      decrement.type,
-      multiply.type
-    ];
-
-    const expected = [
-      'counter/setCounter',
-      'counter/increment',
-      'counter/decrement',
-      'counter/multiply'
-    ];
+      actions: { setCounter, increment, decrement, multiply, randomize }
+    } = createCounterDux();
 
     assert({
-      given: 'autodux is called with args',
-      should: 'produce namespaced action type constants',
-      actual,
-      expected
+      given: "'autodux' is called with 'actions'",
+      should: 'contain action creators with correct action type constants',
+      actual: [
+        setCounter.type,
+        increment.type,
+        decrement.type,
+        multiply.type,
+        randomize.type
+      ],
+      expected: [
+        'counter/setCounter',
+        'counter/increment',
+        'counter/decrement',
+        'counter/multiply',
+        'counter/randomize'
+      ]
     });
   }
-});
 
-describe('autodux().reducer', async assert => {
   {
-    const {
-      actions: { increment, decrement },
-      reducer,
-      initial
-    } = createDux();
+    const { actions } = createCounterDux();
 
-    const actions = [increment(), increment(), increment(), decrement()];
+    const value = 50;
 
     assert({
-      given: 'a reducer',
-      should: 'switch correctly',
-      actual: actions.reduce(reducer, initial),
-      expected: 2
+      given: "'autodux' is called with 'actions'",
+      should:
+        "contain action creator ('set${slice}') for setting the state of the slice",
+      actual: actions.setCounter(value),
+      expected: {
+        type: 'counter/setCounter',
+        payload: value
+      }
     });
   }
 
   {
-    const {
-      actions: { increment, multiply },
+    const { actions, reducer, initial } = createCounterDux();
+
+    const min = 5;
+    const max = 9;
+
+    const counterState = [actions.randomize({ min, max })].reduce(
```

Comment:
Please use a different example.

Reply:
`divide` action creator is now used here.